store/models/service_agreement.go

```diff
@@ -156,6 +152,7 @@ func serviceAgreementIDInputBuffer(encumbrance Encumbrance, digest common.Hash)
 type Encumbrance struct {
 	Payment    *assets.Link   `json:"payment"`
 	Expiration uint64         `json:"expiration"`
+	EndAt      Time           `json:"endAt"`
```

Comment:
I just want it to be explicitly known that we save `EndAt` in three places now: `JobSpecRequest`, `JobSpec`, and `Encumbrance`.